corehq/apps/userreports/views.py

```diff
@@ -861,6 +860,52 @@ def report_source_json(request, domain, report_id):
     return json_response(config)
 
 
+class ExpressionDebuggerView(BaseUserConfigReportsView):
+    urlname = 'expression_debugger'
+    template_name = 'userreports/expression_debugger.html'
+    page_title = ugettext_lazy("Expression Debugger")
+
+
+@login_and_domain_required
+@toggles.USER_CONFIGURABLE_REPORTS.required_decorator()
+def evaluate_expression(request, domain):
+    doc_type = request.POST['doc_type']
```

Comment:
Seems like it'd be convenient to not need to select doc type, but rather have this code attempt to look in the case store, then if that fails try the xform store.